charts/argo-cd/templates/argocd-notifications/servicemonitor.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   name: {{ template "argo-cd.notifications.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
   {{- if .Values.notifications.metrics.serviceMonitor.namespace }}
   namespace: {{ .Values.notifications.metrics.serviceMonitor.namespace }}
   {{- end }}
```

Comment:
I think this should be like below? 🙋 

```diff
- namespace: {{ .Release.Namespace | quote }}
  {{- if .Values.notifications.metrics.serviceMonitor.namespace }}
  namespace: {{ .Values.notifications.metrics.serviceMonitor.namespace }}
+ {{- else }}
+ namespace: {{ .Release.Namespace | quote }}
  {{- end }}
```

Reply:
Yes that's true, thanks for mention that. I've fixed it in https://github.com/argoproj/argo-helm/pull/1937/commits/501d920e168d1b6362e58f0488da20af45a0352a.

Reply:
What about dropping the if/else and use the default func for that?:

```
namespace: {{ default .Release.Namespace .Values.notifications.metrics.serviceMonitor.namespace | quote }}
```

Reply:
I am ok with the proposed solution. Should I implemented?

Reply:
Would appreciate it, yes.
Also please make sure that you update all servicemonitor.yaml for all components, I think you missed some of them